Unix/http/http.c

```diff
@@ -816,13 +817,20 @@ static Http_CallbackResult _WriteHeader(
 
     if (handler->sendPage)
     {
-        buf_size = (size_t)Snprintf(
-            currentLine,
-            sizeof(currentLineBuff),
-            RESPONSE_HEADER_FMT,
-            (int)handler->httpErrorCode,
-            _GetHttpErrorCodeDescription(handler->httpErrorCode),
-            (int)handler->sendPage->u.s.size );
+        if (handler->pSendAuthHeader)
+        {
+            buf_size = (size_t)Snprintf( currentLine, sizeof(currentLineBuff), RESPONSE_HEADER_FMT,
+                                         (int)handler->httpErrorCode, _GetHttpErrorCodeDescription(handler->httpErrorCode),
+                                         (int)handler->sendPage->u.s.size, handler->pSendAuthHeader );
+            PAL_Free(handler->pSendAuthHeader);
+            handler->pSendAuthHeader = NULL;
+        }
+        else 
+        {
+            buf_size = (size_t)Snprintf( currentLine, sizeof(currentLineBuff), RESPONSE_HEADER_FMT,
+                                         (int)handler->httpErrorCode, _GetHttpErrorCodeDescription(handler->httpErrorCode),
+                                         (int)handler->sendPage->u.s.size, "" );
```

Comment:
Nope. The crlf is in the applied string. 

However, this code has been replaced with more legible code that eliminates the sprintfs